### ISSUE TITLE
The pdf viewer does not return to normal size

### ISSUE THREAD

**Issue by mrpau-julius** [learningequality/kolibri#3985]
### Observed behavior
In recommended page, I view the document file name "Decimal". first I download the file, then click the Right arrow(next) in recommended area at the bottom, and click full screen then exit the full screen and the pdf viewer will not return to normal size.

### Expected behavior
The PDF viewer must back to normal size.

### Errors and logs
```
[INFO: kolibri/core/assets/src/core-app/mediator.js] KolibriModule: document_pdf_render_module registered default_frontend-0.11.0a3.js:200:7213
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Will-change memory consumption is too high. Budget limit is the document surface area multiplied by 3 (779328 px). Occurrences of will-change over the budget will be ignored. learn
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12 pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1885564018 (pclr) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unsupported header type 1668112752 (cmap) pdfJSWorker-0.11.0a3.js:1:4452
Warning: Unknown colorspace 12
pdfJSWorker-0.11.0a3.js:1:4452

```

### Steps to reproduce
1. login with admin account.
2. import a token file "Open Stax".
3. then go to recommended page.
4. open any document file to view a PDF.
5. download the file and save it.
6. click the right arrow in bottom twice.
7. click the full screen and then exit the screen to see the changes happen to the PDF viewer.

### Context
windows 7
kolibri 0.11.0a3
firefox 61.0.1(64-bit)

### Screenshot
![erroronrecommended](https://user-images.githubusercontent.com/38421180/42363398-68056124-80ac-11e8-9b2c-5facb3758fcd.gif)

**Comment by indirectlylit** [comment]
@mrpau-julius would you mind checking if this issue exists in 0.10.0 also?

**Comment by mrpau-julius** [comment]
@indirectlylit yes, its also exists after I download the file.

**Comment by indirectlylit** [comment]
cc @christianmemije

**Comment by christianmemije** [comment]
Reproduced in Firefox. Very weird.

**Comment by christianmemije** [comment]
Fixed in #4440
